### DISCUSSION TITLE
screen suspend in stable version 2022 (0.13.0)

### DISCUSSION THREAD

**Discussion by frosstatx** [guysoft/FullPageOS#524] [Q&A]
After 15 minuter in raspberry model 4 in the stable version  enter a suspend mode,  how i can disable this?

**Comment by csabal** [comment]
Are you sure it is your Raspbi? Check your monitor / TV  sleep settings as well.

**Comment by guysoft** [comment]
Testing on another monitor might help. Also assuming 2022 is 0.13.0?

**Comment by smadds** [comment]
I am seeing the same - if I have a mouse plugged into the Pi and move it the screen comes alive again. Not sure if it's going into standby or just screen blanking. I have disabled screen blanking in raspi-config.

**Comment by guysoft** [comment]
I am moving this to a discussion, since it talking about an older version and is not planned to be fixed retrospectively (and does not look like a bug)

**Comment by itsyoshio** [comment]
I have the Same issue on Version 0.13.0.

In my case, i installed the Stable Version of FullPageOS from Raspberry Pi Imager.
This version is, apparenlty, from 2022-03-07
<img width="792" alt="image" src="https://github.com/guysoft/FullPageOS/assets/12760988/f47482cd-ce84-4312-98cc-b38f756a923c">

cat'ing `/etc/fullpageos_version` reveals that it's indeed 0.13.0.

What confused me, ist that 0.13.0 from the releases page is from 2023-10
https://github.com/guysoft/FullPageOS/releases/tag/0.13.0 but still references 

So this issue is present on a Raspberry Pi 3 (non B), with Version 0.13.0.

**Comment by tschlatter** [comment]
If I understand the question correctly, this is not an issue of FullPageOS.

By default, Raspberry Pi has a built in timeout for screen blanking. But you can switch it off:
`sudo raspi-config`

Then select '2 Display Options' followed by 'D2 Screen Blanking'. Set it to 'No'.

**Reply by fapwin** [reply]
Hello, i've the same Problem. 
I am not a linux professional, how do I access the command line in FullPageOS to enter the command?

Many thanks for your help and best regards

**Reply by apartmedia** [reply]
Use either a SSH connection (from another computer) or if you have a display and keyboard connected to the FullPageOS computer, you can switch to a console by pressing ctrl+alt+F1. From there login into the pi and you can issue commands.
ctrl+alt+F7 brings you back to the running X session.

Consider switching to the latest version 0.14.0 (nightly) from the beginning, which is based on the newest Debian bookworm OS. 0.13.0 (stable) is a bit outdated and as far as I understand it @guysoft will not continue to work on the older version.

**Reply by fapwin** [reply]
Hello apartmedia,

Thank you for your reply.

I am now using the latest version 0.14.0.
When I use the console with `sudo raspi-config`, I am supposed to enter a password. `rasbperry` does not work.

Can you tell me how to configure the screen timeout?

With best regards

**Reply by apartmedia** [reply]
Try flipping the „b“ and the „p“ in your password.
It‘s spelled exactly like the fruit…

**Comment by apartmedia** [comment]
Regarding FullPageOS 0.13.0 this could be related to a bash syntax problem in the gui model script [start_gui](https://github.com/guysoft/CustomPiOS/blob/devel/src/modules/gui/filesystem/opt/scripts/start_gui) of CustomPiOS (if-Statement on line 9 and 14).
The latest CustomPiOS start_gui script has fixed this, but it seems that the stable version of FullPageOS (0.13.0 from RPiImager) still uses an old or faulty version of it.

**Reply by guysoft** [reply]
Indeed, I am working to get a new RC out for 0.14.0. What is blocking me now is adding a wpa-supplicant replacement for network manager.

**Reply by apartmedia** [reply]
Totally understand that… the missing wpa-supplicant thing is a mess.
While experimenting with the current 0.14.0 I also found some more ideas and little bugs…

- Not working tab rotating when using FullPageDashboard with multiple addresses (the FullPageDashboard api.php script does in fact not read multiple lines from init.txt (which is altered with the content of fullpagedashboard.txt by the start_chroot_script); it should work if start_chroot_script would create an urls.json with a json encoded content coming from e.g. fullpagedashboard.txt (this could also allow defining the tab entity names used by the dashboard bootstrap html)
- Using Plymouth for splashscreen function instead of the splashscreen.service (to use the standard bookworm splash-system and also allows animated splash screens if needed by the use of scripting)
- Mapping the console to a non-existing fb-driver telling fbcon not to take over the console (as a quick and dirty way to fully suppress the boot messages and only showing the splash screen, later recovering with a call of con2fbmap)

But it‘s understandable that you first wanna update the WiFi stuff before diving into all of these other things…

**Reply by guysoft** [reply]
There is a PR for the NetworkManager settings if you have any input https://github.com/guysoft/CustomPiOS/pull/227

**Comment by ThatCameraOperator** [comment]
Well, this one is a bit daft. The instructions on how to disable the screen blanking/power saver/screen saver (call it whatever you want), should be on the main Setup Instructions page.
Surely someone would have realised that some uses of this OS are for screens that don't have human input devices connected??
